Example/PaymentSheet Example/PaymentSheet Example/ExampleWalletButtonsView.swift

```diff
@@ -427,6 +479,13 @@ class ExampleWalletButtonsModel: ObservableObject {
     func onCompletion(result: PaymentSheetResult) {
         self.addDebugLog("PaymentSheet completion called with result: \(result)")
 
+        // Check if we should simulate payment failure for testing
+        if shopPayTestingOptions.simulatePaymentFailed {
```

Comment:
Is never called in the shopPay flow because this is only hooked up to the 'confirm' button w/ the flow controller.  Seems like in createPaymentIntentWithSPTTestBackend -- any time we set  `self.paymentResult`, we should just call `onCompletion(result:....)`

Reply:
Also on line 315, in the function `preparePaymentSheetWithOriginalBackend` where it says:
```
                        // Create the SPT on your backend here
```

Maybe lets call:
```
                        self?.onCompletion(result: .completed)
```

Reply:
Proposing additional changes here (e.g. a reload button & success) to make success/failure cases in bug bashing more obvious.